### PR TITLE
Feature input

### PR DESCRIPTION
**Pull request by Bhakti199**
created different styles of inputs - label input, error input and Textarea input

### CODE REVIEW

File: input/input.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+@import url(/style.css);
+
+.input-intro {
+  font-size: 1.25rem;
+  padding: 1rem 1rem;
```
Comment (mariyasada) [inline input/input.css]: you can use padding:1rem instead of padding: 1rem 1rem

**Review by mariyasada — APPROVED** [review]
you can merge it

File: input/input.css (hunk 1 above)
Comment (Bhakti199) [reply input/input.css]: okay got it.

File: input/input.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+.form-login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  width: 16rem;
+  height: 20rem;
+  border-radius: 10px;
```
Comment (Manju-sree) [inline input/input.css:20]: try to maintain only one unit through out class.. here for border-radius you can provide in rems

File: input/input.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+.error-input {
+  padding: 0 10px;
+  display: flex;
+  flex-direction: column;
+  gap: 7px;
+}
+.form-input,
+.error-for-password-input {
+  padding: 5px;
+  width: 13rem;
```
Comment (Manju-sree) [inline input/input.css:38]: same here, try to maintain same units

**Review by Manju-sree — CHANGES_REQUESTED** [review]
there are few changes required, other than that everything looks good

File: input/input.css (hunk 3 above)
Comment (Bhakti199) [reply input/input.css:38]: it is less than 10px so gave values in px here

File: input/input.css (hunk 2 above)
Comment (Bhakti199) [reply input/input.css:20]: It is a small px value so I did not converted it to rem.

**Review by Manju-sree — APPROVED** [review]
good to merge, no issues present

**Pull request merged**